config/_default/menus/menus.en.yaml

```diff
@@ -2112,52 +2112,53 @@ main:
     parent: trace_explorer
     weight: 607
   - name: Service Catalog
-    url: tracing/service_catalog/
-    parent: tracing
+    url: service_catalog/
+    pre: service-catalog
+    parent: platform_heading
     identifier: service_catalog
-    weight: 7
+    weight: 110000
   - name: Browsing
-    url: tracing/service_catalog/browsing
+    url: service_catalog/browsing
     parent: service_catalog
     identifier: service_catalog_browse
     weight: 701
```

Comment:
Do you know if these are the standardized weights? Should this be 1? Also, not a blocker, but recommend moving this so that it falls under Alerting